scripts/globals/magic.lua

```diff
@@ -805,7 +807,13 @@ function addBonuses(caster, spell, target, dmg, bonusmab)
 		-- print(mabbonus);
     else
         local mab = caster:getMod(MOD_MATT) + bonusmab;
-        if (spell:getElement() > 0 and spell:getElement() <= 6) then
+
+        local mab_crit = caster:getMod(MOD_MAGIC_CRITHITRATE);
+        if( math.random(1,100) < mab_crit ) then
+           mab = mab + ( 10 + caster:getMod(MOD_MAGIC_CRIT_DMG_INCREASE ) );
+        end
+
+        if (spell:getElement() > 0 and spell:getElement() <= 6 and caster:getMainJob() == JOB_BLM and caster:getMainLvl() >= 75) then

```

Comment:
Just a heads up the get merit function already check to see if the caster is the correct job and of level 75 or higher, so no need to add that check here. I attempted the same change before!